shell/src/main/java/org/apache/accumulo/shell/commands/CreateNamespaceCommand.java

```diff
@@ -87,11 +96,13 @@ public Options getOptions() {
 
     createNamespaceOptCopyConfig =
         new Option("cc", "copy-config", true, "namespace to copy configuration from");
+    createNamespaceOptCopyProperties = new Option("cp", "copy-properties", true,
+        "namespace to copy properties from (only namespace properties are copied)");
```

Comment:
This is so similarly named. Although the naming similarity affects the Java API also, the Java API has the benefit of a decent Javadoc. This could use a better name to avoid getting too verbose help documentation.

For example, instead of creating a separate property (and allowing them both to be used, which could get confusing), just add a `--copy-config-exclude-parent` or just `--exclude-parent` that's used in conjunction with `--copy-config`.

Reply:
c05206a036 changed to use --exclude-parent